packages/core/src/hub.ts

```diff
@@ -35,7 +36,7 @@ import { closeSession, makeSession, updateSession } from './session';
  *
  * @hidden
  */
-export const API_VERSION = 4;
+export const API_VERSION = parseFloat(SDK_VERSION);
```

Comment:
hm, will this actually work?

Reply:
yes it will - the string gets coerced to a float (only taking into account major/minor)